api/src/main/java/gov/cms/ab2d/api/SpringBootApp.java

```diff
@@ -2,9 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-
+import org.springframework.boot.autoconfigure.domain.EntityScan;
 
 @SpringBootApplication
+@EntityScan(basePackages = {"gov.cms.ab2d.domain"})
```

Comment:
`@SpringBootApplication` alone should be sufficient to find the entity classes. is the app not working without EntityScan?

Reply:
Yeah, I believe it's because the entities are in a different directory, if I don't have this it will give an error on startup like:
Caused by: java.lang.IllegalArgumentException: Not a managed type: class gov.cms.ab2d.domain.User
	at org.hibernate.metamodel.internal.MetamodelImpl.managedType(MetamodelImpl.java:552) ~[hibernate-core-5.3.10.Final.jar:5.3.10.Final]

Reply:
👍